apps/native-component-list/App.tsx

```diff
@@ -77,9 +52,11 @@ class App extends React.Component<Props, State> {
           {Platform.OS === 'ios' && <StatusBar barStyle="dark-content" />}
         </View>
       );
-    } else {
+    }
+    if (AppLoading) {
```

Comment:
Would you care to add a comment why we let ourselves return `null` in case `AppLoading` isn't exported from `expo`? This is a situation unique to `bare-expo => ncl`, so for future developers it may come in handy.